src/evm/native.rs

```diff
@@ -111,8 +111,8 @@ impl PrecompiledContract for EcRecover {
         let mut output: Vec<u8> = Vec::new();
         if let Ok(public) = recover(&input, &hash, bit) {
             let data = common::hash::summary(&public.0);
-            output.write_all(&[0; 12])?;
-            output.write_all(&data[12..data.len()])?;
+            output.write_all(&[0; 12]).unwrap();
```

Comment:
Why `unwrap`?

Reply:
The output is a `Vec<u8>`, It will never raise an error.